apps/src/templates/button.module.scss

```diff
@@ -0,0 +1,254 @@
+@import "color.scss";
+
+// Note: Keep these constants in sync with Button.jsx.
+$button-height-default: 34px;
+$button-height-large: 40px;
+$button-height-narrow: 40px;
+$button-height-small: 20px;
+
+div,
+button,
+a,
+a:link,
+a:visited {
+  &.main {
+    display: inline-block;
+    font-size: 12px;
+    font-family: "Gotham 4r", sans-serif;
+    border: 1px solid $border_gray;
+    border-radius: 3px;
+    text-decoration: none;
+    box-sizing: border-box;
+    overflow: hidden;
+    white-space: nowrap;
+
+    &:hover {
+      background-color: $white;
+      cursor: pointer;
+    }
+
+    &:disabled,
+    &[disabled] {
```

Comment:
had to add `[disabled]` to the selectors here, because only inputs like `<button>` are valid with the `:disabled` selector. since the Button component can be rendered as `div` or `a`, we have to use e.g. `div[disabled]` to select the expected elements.